models/demos/falcon40b/demo/demo.py

```diff
@@ -591,7 +591,7 @@ def test_demo(
     model_location_generator,
     get_tt_cache_path,
     all_devices,
-    use_program_cache,
+    # use_program_cache, # TODO: remove workaround for avoiding PCC issues when fixed
```

Comment:
Do we have to turn off program cache? Also, if we have this, we don't need the manual enable in the demo code.